hack/.golint_failures

```diff
@@ -605,6 +605,8 @@ staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf
 staging/src/k8s.io/code-generator/cmd/lister-gen/generators
 staging/src/k8s.io/component-base/cli/flag
 staging/src/k8s.io/component-base/config/v1alpha1
+staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1
+staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1
```

Comment:
where are these from?

Reply:
SetDefault_... that is added to those two packages fails golint.